docs/hiring-guide/community-spotlight/interview-celeste-horgan.rst

```diff
@@ -0,0 +1,85 @@
+    BlackBerry team on their Java SDK. 
+    
+    I was one of 600 co-ops they took in that semester. Because they took in so many, they had a
+    pretty rigorous training program in every department. I was one of about 5 or 10 in a 40 person technical writing department. They
+    put us all through a training program where they taught us what DITA was. They taught us about single source authoring. And, yeah,
+    that's kind of where I started. After I graduated university, I worked as a web designer for a year or two, but it was mostly 
+    stubbornness on my part. Because I had done this degree I was like, "No, I'm gonna do it! I'm gonna go be a designer!" But I
+    really didn't like it. My initial thoughts when I was in school were correct. It just wasn't that interesting to me. So I switched
+    back into technical writing in 2016, and that's what I've been doing ever since. I'm at my second job now as a technical writer.
+    Since then I've worked at a company called Elastic Path, in Vancouver. And now I'm in Berlin at a company called commercetools.
+
+**What was it like searching for jobs outside of Canada?**
+
+    You know what? It was almost comically easy. I kind of just dropped my resume in people's inboxes, using the forms on their
+    websites. I wasn't going out of my way to network or anything, and there was a surprising amount of interest. And I was applying
+    quite globally. I applied for a few places in the U. S. and got responses. I think technical writing is kind of a funny of job in
+    that I think technical writers in organizations often feel very undervalued. But in my last job, and in my current job, I've
+    actually done quite a bit of interviewing and quite a bit of hiring. And it's so funny, because you see so many people who just
+    don't seem to have a lot of self-confidence or passion. So if you're even a little bit passionate or a little opinionated, or you
+    kind of just got something going on, you stand out in an interview so quickly. And people are willing to take you abroad for that,
+    which is really cool.
+
+**What is it like working abroad as a technical writer?**
+
+    Yeah, it's interesting as a writer especially. When you're in North America, in Canada and the United States, the people that you
+    work with are native speakers. So when you're having a conversation about ensuring that a sentence communicates the essence of
+    what was implemented technically, you are quite literally both speaking the same language. If you as a writer decided to reword
+    something using simpler words, maybe using present tense, your subject matter expert or developer that you're working with knows
+    enough English to say, "No, that's not what I meant at all." So it's a It's a high level conversation. 
+    
+    And commercetools is a verydiverse company. For most of the people that I work with English is not their first language. It might
```

Comment:
```suggestion
    And commercetools is a very diverse company. For most of the people that I work with English is not their first language. It might
```

Spacing.